packages/bot-web-ui/src/stores/toolbar-store.js

```diff
@@ -103,7 +103,7 @@ export default class ToolbarStore {
             },
             'reset'
         );
-
+        this.root_store.run_panel.onStopButtonClick();
```

Comment:
We should show a dialog and ask the user for confirmation before resetting + stopping the bot.